src/legacy/status_im/data_store/communities.cljs

```diff
@@ -48,6 +48,17 @@
   [token-permission]
   (= (:type token-permission) constants/community-token-permission-become-member))
 
+(defn- <-images-rpc
```

Comment:
@qfrank, my initial implementation in this PR is the safest one to avoid regressions, but it means I had to transform the response from status-go into the compatible code we use in mobile.

This is what gets returned now from status-go.

```clojure
{:banner    "https://Localhost:46321/...",
 :large     "https://Localhost:46321/...",
 :thumbnail "https://Localhost:46321/..."}
```

We transform to

```clojure
{:banner    {:uri "https://Localhost:46321/..."},
 :large     {:uri "https://Localhost:46321/..."},
 :thumbnail {:uri "https://Localhost:46321/..."}}
```

If we could change status-go branch to keep the value of each type of image as a map with at least the key `uri`, then it would fully work out of the box in mobile without any extra work (and perf penalty) and maybe require less changes in desktop too. 

If you think this is not the best way to go I can remove this function, but then I will need to adjust quite a few more places in this PR. Please, let me know what you prefer :)

Reply:
Update client, and all working well @qfrank